my-app/src/components/ApplicationViews.js

```diff
@@ -1,10 +1,17 @@
-import React from "react"
+import React, { useEffect, useState } from "react"
 import { withRouter, Route } from "react-router-dom"
 import Login from "./auth/Login"
 import Register from "./auth/Register"
 import Blank from "./blank"
+import Product from "./product/ProductDetail"
+import APImanager from "../modules/APImanager"
 
 const ApplicationViews = () => {
+  const [products, setProducts] = useState([])
+
+  APImanager.getAll(products)
```

Comment:
needs to be in a function called getProducts which will evoke setProducts as its last step.
eg:     ```
const getParkAreas = () => {
        if (isAuthenticated()) {
            fetch('http://localhost:8000/parkareas', {
                "method": "GET",
                "headers": {
                    "Accept": "application/json",
                    "Authorization": `Token ${localStorage.getItem("kennywood_token")}`
                }
            })
                .then(response => response.json())
                .then(setAreas)
        }
    }

    useEffect(getParkAreas, [])
```